applications/SwimmingDEMApplication/python_scripts/swimming_DEM_solver.py

```diff
@@ -248,8 +256,9 @@ def SolveDEM(self):
 
         alpha = 1.0 - (self.next_time_to_solve_fluid - self.time) / self.fluid_dt
 
-        if it_is_time_to_forward_couple or self.first_DEM_iteration:
-            self.ApplyForwardCoupling(alpha)
+        if not self.fluid_solver.settings.Has('time_integration_settings'):
```

Comment:
idem, use and operator and try to read options like this one that will be called many times and just store them in the class or provide a Get...() method to obtain them